src/components/link/link.hbs

```diff
@@ -5,10 +5,15 @@
   LICENSE file in the root directory of this source tree.
 -->
 
-<a href="#" class="{{@root.prefix}}--link">Link</a>
-<a class="{{@root.prefix}}--link">Placeholder link</a>
+<div style="margin-bottom: 32px">
```

Comment:
Is this demo-only element/style? If so, can we add styles to `demo.scss` with `.component-example__live--rendered.link` selector...?